### PR TITLE
Spanish: Add first translation, Introduction to the Learning Path

### PR DESCRIPTION
**Pull request by dicortazar**

### CODE REVIEW

**Comment by dicortazar** [comment]
This is a work in progress with the goal of checking if this is the right approach and if you any advise.

Thanks!

**Comment by lenucksi** [comment]
> 
> 
> This is a work in progress with the goal of checking if this is the right approach and if you any advise.
> 
> Thanks!

Looks good so far as I'm concerned. @tsadler1988 might want to advice on anything we need to look after for our rendering things.
Also, consider adding a bunch of translation strings to #333

**Comment by rrrutledge** [comment]
Awesome, Daniel!  I myself speak fluent Spanish and so could review, but we'd prefer a native speaker.  Do you know of one in the InnerSource Commons community?  Ideally a non-Bitergian.

**Comment by tsadler1988** [comment]
Thanks for this @dicortazar! 

Regarding rendering on the website, you will need to add `es` to the [Introduction translations config](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/scripts/generate_learning_path_markdown.js#L44) as per the [instructions for updating innersourcecommons.org with new articles](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/scripts/generate_learning_path_markdown.js#L44), but this isn't mandatory and could be done separately.

**Review by rrrutledge — APPROVED** [review]

File: introduction/es/01-introduction-es.asciidoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Este plan de aprendizaje es una introducción a InnerSource.
+InnerSource consiste en la aplicación de las mejores prácticas de desarrollo y principios de trabajo de las comunidades de software libre dentro de una corporación.
+El software desarrollado no es por lo tanto software libre, pero sí que está en abierto y accesible dentro de los confines de la organización para que cualquier persona pueda usarlo y contribuir a su desarrollo.
+Esta estrategia permite que se abra la posibilidad de colaboración de manera amplia y efectiva, produciendo software que evoluciona y se modifica de forma ágil según las necesidades de los diferentes clientes internos.
+
+Aquí aprenderás a reconocer proyectos y situaciones que son buenos candidatos para InnerSource.
+Mostraremos de un vistazo y a alto nivel cómo InnerSource puede ser útil y ayudar en estas situaciones.
+Te familiarizarás con el vocabulario y términos comunes cuando hablemos de InnerSource.
+Además, enumeraremos los principios clave en los que se basa InnerSource y los beneficios que se obtienen cuando se aplica de manera efectiva.
```
[the remark this answers is not in the document]
Comment (rrrutledge) [reply introduction/es/01-introduction-es.asciidoc:11]: Actually it's not a typo.

**Review by rrrutledge — CHANGES_REQUESTED** [review]
!Gracias por este trabajo!

File: introduction/es/02-problems-solved-es.asciidoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  Esta opción no requiere ningún tipo de esfuerzo extra.
+  Dependiendo del beneficio que traiga la nueva funcionalidad, puede que esperar sea suficiente.
+  Sin embargo esta situación puede frustrar y traer problemas en el medio o largo plazo si esa funcionalidad nunca se lleva a cabo.
+. *Buscar otra solución*. El equipo que necesita esa funcionalidad no puede o no quiere esperar y realiza trabajo extra para compensar la ausencia de dicha mejora.
+  Este trabajo extra puede que traiga cambios en el equipo que usa el software.
+  Alternativamente, el equipo consumidor puede crear un nuevo proyecto que se adecúe a sus necesidades y reemplace el uso de la herramienta original en parte o totalmente (duplicación de código/proyecto y esfuerzos).
+  Esta estrategia permite que el equipo que consume el software obtenga su nueva funcionalidad deseada a través de sus propios esfuerzos, aunque esto venga con ciertos inconvenientes.
+ ..Cualquier tipo de trabajo que realice el equipo que consume el software no va a poder ser reutilizado por otros equipos que necesiten la misma funcionalidad.
```
[the remark this answers is not in the document]
Comment (dicortazar) [reply introduction/es/02-problems-solved-es.asciidoc:21]: Yes, I copied this from the original document format.

Should I use stars as you suggest?

File: introduction/es/02-problems-solved-es.asciidoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ ..El equipo consumidor acaba de firmar un contrato no escrito donde van a tener que mantener dicho código en el largo plazo, lo cual no es parte de sus funciones y quizá ni siquiera sea parte de sus competencias.
```
[the remark this answers is not in the document]
Comment (dicortazar) [reply introduction/es/02-problems-solved-es.asciidoc:22]: This makes sense, thanks!

File: introduction/es/02-problems-solved-es.asciidoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+Este enfoque es diferente respecto a lo visto hasta ahora en organizaciones con estructuras más tradicionales donde ideas y producto se generan en forma de silos y quedan limitadas por la jerarquía corporativa. Vamos a explorar un ejemplo de esta situación.
+
+
+Imagina dos equipos de la misma compañía que producen dos productos finales diferentes donde uno depende del otro.
+Un ejemplo podría ser una aplicación final de usuario que depende de una API que recoge datos que después son visualizados. Esta situación es en realidad relativamente común dentro de las grandes corporaciones, donde un solo equipo produce un software que puede tener docenas o centenares de usuarios finales.
+
+Cuando esos consumidores finales necesitan añadir funcionalidades, los equipos que desarrollan generalmente tienen procesos internos donde priorizan y gestionan los requisitos y así deciden en qué funcionalidades trabajarán.
+En el caso de funcionalidades críticas para los equipos que consumen ese software hay generalmente tres opciones, aunque cada una tiene sus propios problemas.
```
[the remark this answers is not in the document]
Comment (dicortazar) [reply introduction/es/02-problems-solved-es.asciidoc:11]: You're right that the very last sentence is not exactly the same as the one in English. I'll reword this a bit :).

File: introduction/es/02-problems-solved-es.asciidoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (dicortazar) [reply introduction/es/02-problems-solved-es.asciidoc]: Fixed in the new version

File: introduction/es/02-problems-solved-es.asciidoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (dicortazar) [reply introduction/es/02-problems-solved-es.asciidoc]: Fixed in the new version

File: introduction/es/02-problems-solved-es.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (rrrutledge) [reply introduction/es/02-problems-solved-es.asciidoc:21]: Oh, you're right!  Something's wrong, though. It doesn't render as a list.

Screen Shot 2021-02-19 at 6 13 40 AM

Comparing again to the English version, perhaps you need a space between the dots and the text?

```suggestion
 .. Cualquier tipo de trabajo que realice el equipo que consume el software no va a poder ser reutilizado por otros equipos que necesiten la misma funcionalidad.
```

**Review by rrrutledge — COMMENTED** [review]
Looking really good!  Just one thing I noticed.

File: introduction/es/02-problems-solved-es.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (dicortazar) [reply introduction/es/02-problems-solved-es.asciidoc]: Fixed :)

**Comment by dicortazar** [comment]
I think we have covered all the suggestions @rrrutledge :). Thanks for all of your comments and pointers.

**Comment by rrrutledge** [comment]
Looking good!  I will try to get this merged and deployed before our Thursday meeting.

**Review by rrrutledge — APPROVED** [review]

**Pull request merged**